cli/lsp/language_server.rs

```diff
@@ -987,17 +987,16 @@ impl Inner {
     let asset_or_document = self.get_cached_asset_or_document(&specifier)?;
     let line_index = asset_or_document.line_index();
 
-    let req = tsc::RequestMethod::GetNavigationTree(specifier);
-    let navigation_tree: tsc::NavigationTree = self
-      .ts_server
-      .request(self.snapshot()?, req)
-      .await
-      .map_err(|err| {
-        error!("Failed to request to tsserver {}", err);
-        LspError::invalid_request()
+    let navigation_tree =
```

Comment:
Not directly related to fixing the issue, discovered that we were not using the potentially cached version of the navtree here, which is not good.